### DISCUSSION TITLE
unable to deploy

### DISCUSSION THREAD

**Discussion by theabhayprajapati** [vercel/vercel#7796] [Help] [answered]
[`Repo Link`](https://github.com/theabhayprajapati/ai-car)

I was trying to deploy this repository on VERCEL which is having HTML CSS and plain JavaScript file,the deployment was successful, however I am unable to see my project in the URL

404 not found ❌

[`URL`](https://aicar.vercel.app)

**Comment by swarnava** [comment, marked as answer]
Hi, you can access it using https://aicar.vercel.app/main.html. Please setup a redirect using `vercel.json` to serve this as your root url. Thanks!
